app/src/main/java/com/imashnake/animite/dev/ext/MediaSeason.kt

```diff
@@ -23,3 +23,9 @@ fun MediaSeason.nextSeason(now: LocalDate): Pair<MediaSeason, Int> {
         else -> MediaSeason.UNKNOWN__ to 0
     }
 }
+
+/**
+ * Returns a capitalized [MediaSeason.rawValue].
+ */
+val MediaSeason.string
```

Comment:
It might make sense to make a `val label` or `val title` inside MediaSession, rather than an extension
Or maybe we can even override toString for this

Reply:
We can't override toString (or make the variables) since the class is generated no?

Reply:
Make a sanitised model with this infor then 🗡️

Reply:
*Not now* (I'll do it when I refactor `MediaPage`).